option optimize_for = LITE_RUNTIME;

message GallantRaidInfoAnswer
{
 required uint32 result = 1;
 required uint32 id = 2;
 required uint32 refresh = 3;
 required uint32 fight = 4;
}
